src/views/EditEntries/index.js

```diff
@@ -295,21 +299,42 @@ export default class EditEntries extends React.PureComponent {
 
     // PERMISSION
 
-    setEntryData = (val) => {
-        console.warn('Changing entry');
-        this.props.setEntryData(val);
-    }
-
     setExcerpt = (val) => {
-        console.warn('Changing excerpt');
+        if (this.shouldDisableEntryChange(val.id)) {
+            console.warn('No permission to edit entry excerpt');
+            return;
+        }
         this.props.setExcerpt(val);
     }
 
+    setEntryData = (val) => {
+        if (this.shouldDisableEntryChange(val.id)) {
+            console.warn('No permission to edit entry');
```

Comment:
Shouldn't you notify the user instead of the browser

Reply:
notify garina, harek letter type garda euta notification aucha natra

Reply:
Ramro ho ni tyo tah